src/core/WallGraphicsScene.h

```diff
@@ -66,13 +66,14 @@ class WallGraphicsScene : public QGraphicsScene
 private:
     void drawBackground( QPainter* painter, const QRectF &rect ) override;
 
-    QColor backgroundColor_;
-    QList<RenderablePtr> renderables_;
-
     void clear( const QColor& clearColor );
     void setOrthographicView( const QRectF& rect );
 
-public:
+    QColor backgroundColor_;
+    QList<RenderablePtr> renderables_;
+
+    // For text rendering with Qt functionss
+    friend class RenderContext;
```

Comment:
I don't see a simple way right now (it was public in the previous commit ;-)). Maybe take a pointer to the render context and have setters there for these two values, but is it worth the effort..? This text rendering is a big hack of course, which must go away once we move the rendering  away from drawBackground() to QML.